docs/develop/trusted-domains.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Wildcard trusted domains
+description: Learn how to use wildcards to specify trusted domains that aren't listed in the manifest.
+ms.date: 11/14/2023
+ms.localizationpriority: medium
+---
+
+# Wildcard trusted domains
+
+Besides its own domain, an add-in can access resources in certain other domains such as login points for major identity providers and in any domain listed in the manifest. The latter domains are specified in the [AppDomains](/javascript/api/manifest/appdomains) element of the XML manifest or the [validDomains](/microsoftteams/platform/resources/schema/manifest-schema-dev-preview#validdomains) property of the unified manifest. Wildcards aren't allowed in the XML manifest. They are allowed in the unified manifest because some Teams apps and other Microsoft 365 apps honor them; but Office Add-ins don't honor "validDomains" that contain wildcards. 
```

Comment:
the style guide recommends that we not use the term "login" but instead should use some form of "sign in"

Reply:
I'm having GitHub issues, but I'll get this fixed today.